source/guide_codimd.rst

```diff
@@ -0,0 +1,398 @@
+
+  * MySQL_
+  * supervisord_
+  * domains_
+  * Node_ and its package manager npm_
+
+
+Prerequisites
+=============
+
+.. include:: includes/my-print-defaults.rst
+
+We're using Node_ in the stable version 8:
+
+::
+
+ [isabell@stardust ~]$ uberspace tools version show node
+ Using 'Node.js' version: '8'
+ [isabell@stardust ~]$
+
+Your blog URL needs to be setup:
+
+.. include:: includes/web-domain-list.rst
+
+Installation
+============
+
+You need a special database for CodiMD_:
+
+.. warning:: Replace ``<username>`` with your username!
+
+Create Database
+---------------
+
+.. code-block:: console
+
+  [isabell@stardust ~]$ mysql -e "CREATE DATABASE <username>_codimd CHARACTER SET utf8 COLLATE utf8_general_ci;"
+  [isabell@stardust ~]$
+
+Download Sources
+----------------
+
+go to the release_-site and download the latest version 
+
+.. code-block:: console
+
+  [isabell@stardust ~]$ wget https://github.com/hackmdio/codimd/archive/6.6.66.zip
+  --2018-10-17 18:50:29--  https://github.com/hackmdio/codimd/archive/6.6.66.zip
+  Resolving github.com (github.com)... 192.30.253.112, 192.30.253.113
+  Connecting to github.com (github.com)|192.30.253.112|:443... connected.
+  HTTP request sent, awaiting response... 302 Found
+  Location: https://codeload.github.com/hackmdio/codimd/zip/1.2.1 [following]
+  --2018-10-17 18:50:29--  https://codeload.github.com/hackmdio/codimd/zip/1.2.1
+  Resolving codeload.github.com (codeload.github.com)... 192.30.253.121, 192.30.253.120
+  Connecting to codeload.github.com (codeload.github.com)|192.30.253.121|:443... connected.
+  HTTP request sent, awaiting response... 200 OK
+  Length: unspecified [application/zip]
+  Saving to: ‘6.6.66.zip’
+  
+      [     <=>                                                                                                                                        ] 7,090,453   6.28MB/s   in 1.1s   
+  
+  2018-10-17 18:50:31 (6.28 MB/s) - ‘6.6.66.zip’ saved [7090453]
+  [isabell@stardust ~]$ unzip 6.6.66.zip
+  [...]
+  [isabell@stardust ~]$ mv codimd-6.6.66/ codimd
+  [isabell@stardust ~]$ rm 6.6.66.zip
+  [isabell@stardust ~]$ cd codimd
+  [isabell@stardust codimd]$ bin/setup
+  copy config files
+  install npm packages
+  [...]
+  [isabell@stardust codimd]$
+
+
+Remove UglifyJS
+---------------
+
+The UglifyJS plugin freezes the compilation step, because it tries to compile everything in parallel and then reaches the maximum memory-limit/user of uberspace. And since it's apparently not necessary, we'll just remove it.
+
+Open ``webpack.production.js`` and comment out this line (currently line 6):
+
+.. code-block:: javascript
+
+  var ParallelUglifyPlugin = require('webpack-parallel-uglify-plugin')
+
+So that it looks like this:
+
+.. code-block:: javascript
+  :emphasize-lines: 1
+
+  //var ParallelUglifyPlugin = require('webpack-parallel-uglify-plugin')
+
+
+And comment out this block (currently line 15 - 26):
```

Comment:
Please explain why

Reply:
Doesn't work
```
[testnico@machholz codimd]$ npm run build
npm WARN lifecycle The node binary used for scripts is /bin/node but npm is using /opt/nodejs8/bin/node itself. Use the `--scripts-prepend-node-path` option to include the path for the node binary npm was executed with.

> CodiMD@1.2.1 build /home/testnico/codimd
> webpack --config webpack.production.js --progress --colors --bail

/home/testnico/codimd/webpack.production.js:15
    /*new ParallelUglifyPlugin({
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^

SyntaxError: Invalid or unexpected token
    at createScript (vm.js:80:10)
    at Object.runInThisContext (vm.js:139:10)
    at Module._compile (module.js:616:28)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at module.exports (/home/testnico/codimd/node_modules/webpack/bin/convert-argv.js:80:13)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! CodiMD@1.2.1 build: `webpack --config webpack.production.js --progress --colors --bail`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the CodiMD@1.2.1 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/testnico/.npm/_logs/2018-10-24T21_08_55_763Z-debug.log
```

Reply:
can you paste the output of the first 35 lines of your `webpack.production.js`?